### PR TITLE
feat: add replay functionality

### PR DESCRIPTION
**Pull request by rach-id**
## Overview
Closes https://github.com/celestiaorg/blobstream-ops/issues/2

### CODE REVIEW

File: cmd/blobstream-ops/replay/config.go [hunk 1]
```diff
@@ -0,0 +1,215 @@
...
+		return fmt.Errorf("the EVM address cannot be empty")
+	}
+	if !ethcmn.IsHexAddress(addr) {
+		return errors.New("valid EVM address is required")
+	}
+	return nil
+}
+
+func parseFlags(cmd *cobra.Command) (Config, error) {
+	// TODO add support for env variables
```
Comment (rach-id) [inline cmd/blobstream-ops/replay/config.go:107]: Support env variables

File: cmd/blobstream-ops/replay/config.go [hunk 2]
```diff
@@ -0,0 +1,215 @@
...
+		return Config{}, err
+	}
+
+	verify, err := cmd.Flags().GetBool(FlagVerify)
+	if err != nil {
+		return Config{}, err
+	}
+
+	// TODO add rate limiting flag
+	// TODO add gas price multiplier flag
```
Comment (rach-id) [inline cmd/blobstream-ops/replay/config.go:199]: support gas multiplier flag

File: cmd/blobstream-ops/replay/config.go [hunk 3]
```diff
@@ -0,0 +1,215 @@
...
+	if err != nil {
+		return Config{}, err
+	}
+
+	verify, err := cmd.Flags().GetBool(FlagVerify)
+	if err != nil {
+		return Config{}, err
+	}
+
+	// TODO add rate limiting flag
```
Comment (rach-id) [inline cmd/blobstream-ops/replay/config.go:198]: support rate limiting for EVM endpoint

**Review by evan-forbes — APPROVED** [review]
:+1:

**Pull request merged**